### ISSUE TITLE
I need validation loss plot

### ISSUE THREAD

**Issue by MheadHero** [sovit-123/fasterrcnn-pytorch-training-pipeline#32]

**Comment by sovit-123** [comment]
Will plan on adding it. Have it working on some other important things like aspect ratio resizing and mosaic similar to Ultralytics YOLOv5. Next, I will try to target this.

**Comment by NammaThalle** [comment]
is this ready? or can you give any work around for this for now?

**Comment by sovit-123** [comment]
@NammaThalle 
I was trying to figure this out. But the only way available introduces some duplicate code in the validation loop and also leads to repeating the validation forward pass to calculate the mAP. So, this feature is on hold for now.

**Comment by EmmaVanPuyenbr** [comment]
I have the same question :)

**Comment by sovit-123** [comment]
@EmmaVanPuyenbr I will try to figure it out but cannot promise anything.
